tracing-subscriber/src/fmt/writer.rs

```diff
@@ -103,6 +104,60 @@ impl MakeWriter for TestWriter {
     }
 }
 
+/// A writer that erases the specific [`io::Write`] and [`Makewriter`] types being used.
+///
+/// This is useful in cases where the concrete type of the writer cannot be known
+/// until runtime.
+///
```

Comment:
Nit, take it or leave it: It would be nice to have example(s) here showing how this type can be used?

Reply:
Done 👍